src/service/grpcweb.ts

```diff
@@ -222,13 +222,13 @@ function generateTypescriptDefinition(fileDescriptor: FileDescriptorProto, expor
   printer.printIndentedLn(`on(type: 'end', handler: () => void): RequestStream<T>;`);
   printer.printIndentedLn(`on(type: 'status', handler: (status: Status) => void): RequestStream<T>;`);
   printer.printLn(`}`);
-  printer.printLn(`interface BidirectionalStream<T> {`);
-  printer.printIndentedLn(`write(message: T): BidirectionalStream<T>;`);
+  printer.printLn(`interface BidirectionalStream<Req, Res> {`);
```

Comment:
I'd like to know opinions on the Generic type variables here, but I can't use `<R, R>`, and I think `<T, V>` is just unclear.

Reply:
Maybe `<ReqT, ResT>`?

Reply:
Yep ReqT and ResT work for me